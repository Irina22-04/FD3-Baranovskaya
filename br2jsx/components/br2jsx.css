div {
    background-color: darkslategrey;
    color: white;
    text-align: center;
    width: max-content;
    padding: 20px;
    margin-left: auto;
    margin-right: auto;
    font-size: 18px;
}