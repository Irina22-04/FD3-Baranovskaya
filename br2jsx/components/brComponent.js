import React from 'react';
import PropTypes from 'prop-types';

class BrComponent extends React.Component {

    static propTypes = {
        text: PropTypes.string.isRequired,
    };

    render() {

        return (
            <React.Fragment>
            {this.props.text}
            <br />
            </React.Fragment>
    );
    }
}

export default BrComponent;