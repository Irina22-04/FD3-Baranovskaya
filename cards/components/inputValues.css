.errMessage {
    display: inline-block;
    width: max-content;
    color: red;
    left: 350px;
    position: absolute;
}

.userInput {
    height: 30px;
}

input {
    position: absolute;
    left: 180px;
}

input[data-main] {
    left: 40px;
}

label {
    font-size: 20px;
}