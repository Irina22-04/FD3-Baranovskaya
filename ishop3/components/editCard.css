input {
    width: 30%;
}

.card {
    font-size: 20px;
}

.cardTitle {
    font-size: 25px;
    font-weight: bold;
    margin-bottom: 20px;
}

.card input {
    position: absolute;
    left: 10%;
    font-size: 18px;
}

.controlEdit:last-child {
    transform: translateY(100%);
}

.errDiv {
    color: red;
}