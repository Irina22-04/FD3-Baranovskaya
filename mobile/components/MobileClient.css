div.mobileClient
{
  padding: 2px 0;
  display: flex;
}

.clientSurname,
.clientName,
.clientPatronym,
.clientStatus,
.clientDelete{
  width: 13%;
}

.clientEdit {
  width: 16%;
}

.clientBalance {
  width: 10%;
}

.red {
  background-color: red;
}

.green {
  background-color: green;
}


