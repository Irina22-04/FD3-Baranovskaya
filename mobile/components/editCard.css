.edit-card {
    font-size: 16px;
    margin-top: 50px;
}

.edit-card input {
    position: absolute;
    left: 12%;
    font-size: 14px;
    width: 30%;
    padding-bottom: 0;
}

.edit-card .controlEdit {
    width: 70px;
    transform: translateY(10px);
}

.controlEdit:last-child {
    transform: translate(80px, 10px);
}