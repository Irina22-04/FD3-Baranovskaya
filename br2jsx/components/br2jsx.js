import React from 'react';
import PropTypes from 'prop-types';

import './br2jsx.css';


class Br2jsx extends React.Component {

    static propTypes = {
        text: PropTypes.string.isRequired,
    };

    render() {

        const arrayText = this.props.text.split(/<br\s?\/?>/g,);
        const dataLength = arrayText.length;
        const result = [];

        arrayText.forEach((item, index) => {
            result.push(item);
            if (index !== dataLength - 1) {
                result.push(<br/>)
            }
        });

        return (
            <div className={'br2jsx'}>
                {result}
            </div>
        );
    }
}

export default Br2jsx;