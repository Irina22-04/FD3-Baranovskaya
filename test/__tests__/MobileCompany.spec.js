"use strict";

import React from 'react';
import renderer from 'react-test-renderer';

import MobileCompany from '../components/MobileCompany';
import {cardEvents} from '../components/editCard';
import {clientEvents} from '../components/MobileClient';

const clientsArr = [
    {id: 101, fam: "Иванов", im: "Иван", otch: "Иванович", balance: 200},
    {id: 105, fam: "Сидоров", im: "Сидор", otch: "Сидорович", balance: 250},
    {id: 110, fam: "Петров", im: "Пётр", otch: "Петрович", balance: 180},
    {id: 120, fam: "Григорьев", im: "Григорий", otch: "Григорьевич", balance: -220},
];

describe('MobileCompany', () => {
    const component = renderer.create(
        <MobileCompany clients={clientsArr}/>
    );

    let componentTree = component.toJSON();
    expect(componentTree).toMatchSnapshot();

    it('show blocked client should work', () => {

        const buttonBlockedElem = component.root.findByProps({className: "blocked"});
        buttonBlockedElem.props.onClick();

        expect(component.getInstance().state.clients).toEqual([{
            id: 120,
            fam: 'Григорьев',
            im: 'Григорий',
            otch: 'Григорьевич',
            balance: -220
        }]);

        componentTree = component.toJSON();
        expect(componentTree).toMatchSnapshot();
    });

    it('show active client should work', () => {

        const buttonActiveElem = component.root.findByProps({className: "active"});
        buttonActiveElem.props.onClick();

        expect(component.getInstance().state.clients).toEqual([
            {
                id: 101,
                fam: 'Иванов',
                im: 'Иван',
                otch: 'Иванович',
                balance: 200
            },
            {
                id: 105,
                fam: 'Сидоров',
                im: 'Сидор',
                otch: 'Сидорович',
                balance: 250
            },
            {
                id: 110,
                fam: 'Петров',
                im: 'Пётр',
                otch: 'Петрович',
                balance: 180
            }
        ]);

        componentTree = component.toJSON();
        expect(componentTree).toMatchSnapshot();
    });

    it('show all client should work', () => {

        const buttonAllElem = component.root.findByProps({className: "all"});
        buttonAllElem.props.onClick();

        expect(component.getInstance().state.clients).toEqual([
            {
                id: 101,
                fam: 'Иванов',
                im: 'Иван',
                otch: 'Иванович',
                balance: 200
            },
            {
                id: 105,
                fam: 'Сидоров',
                im: 'Сидор',
                otch: 'Сидорович',
                balance: 250
            },
            {
                id: 110,
                fam: 'Петров',
                im: 'Пётр',
                otch: 'Петрович',
                balance: 180
            },
            {
                id: 120,
                fam: 'Григорьев',
                im: 'Григорий',
                otch: 'Григорьевич',
                balance: -220
            }
        ]);

        componentTree = component.toJSON();
        expect(componentTree).toMatchSnapshot();
    });

    it('edit client should work', () => {
        cardEvents.emit('edit', {id: 101, fam: 'Иванов', im: 'Иван', otch: 'Иванович', balance: 350});
        expect(component.getInstance().state.clients).toEqual([
            {
                id: 101,
                fam: 'Иванов',
                im: 'Иван',
                otch: 'Иванович',
                balance: 350
            },
            {
                id: 105,
                fam: 'Сидоров',
                im: 'Сидор',
                otch: 'Сидорович',
                balance: 250
            },
            {
                id: 110,
                fam: 'Петров',
                im: 'Пётр',
                otch: 'Петрович',
                balance: 180
            },
            {
                id: 120,
                fam: 'Григорьев',
                im: 'Григорий',
                otch: 'Григорьевич',
                balance: -220
            }
        ]);

        componentTree = component.toJSON();
        expect(componentTree).toMatchSnapshot();
    });

    it('add client should work', () => {
        const buttonAddClientElem = component.root.findByProps({className: "addClient"});
        buttonAddClientElem.props.onClick();

        cardEvents.emit('edit', {id: 121, fam: 'Антонов', im: 'Антон', otch: 'Антонович', balance: 350});
        expect(component.getInstance().state.clients).toEqual([
            {
                id: 101,
                fam: 'Иванов',
                im: 'Иван',
                otch: 'Иванович',
                balance: 350
            },
            {
                id: 105,
                fam: 'Сидоров',
                im: 'Сидор',
                otch: 'Сидорович',
                balance: 250
            },
            {
                id: 110,
                fam: 'Петров',
                im: 'Пётр',
                otch: 'Петрович',
                balance: 180
            },
            {
                id: 120,
                fam: 'Григорьев',
                im: 'Григорий',
                otch: 'Григорьевич',
                balance: -220
            },
            {
                id: 121,
                fam: 'Антонов',
                im: 'Антон',
                otch: 'Антонович',
                balance: 350
            }
        ]);

        componentTree = component.toJSON();
        expect(componentTree).toMatchSnapshot();
    });

    it('delete client should work', () => {
        clientEvents.emit('deleteClient', 121);
        expect(component.getInstance().state.clients).toEqual(
            [
                {
                    id: 101,
                    fam: 'Иванов',
                    im: 'Иван',
                    otch: 'Иванович',
                    balance: 350
                },
                {
                    id: 105,
                    fam: 'Сидоров',
                    im: 'Сидор',
                    otch: 'Сидорович',
                    balance: 250
                },
                {
                    id: 110,
                    fam: 'Петров',
                    im: 'Пётр',
                    otch: 'Петрович',
                    balance: 180
                },
                {
                    id: 120,
                    fam: 'Григорьев',
                    im: 'Григорий',
                    otch: 'Григорьевич',
                    balance: -220
                }
            ]
        );

        componentTree = component.toJSON();
        expect(componentTree).toMatchSnapshot();
    })

});