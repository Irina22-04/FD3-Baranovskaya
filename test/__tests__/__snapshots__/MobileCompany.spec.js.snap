// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div
    className="mobileCompany"
  >
    <div
      className="buttons"
    >
      <input
        className="all"
        onClick={[Function]}
        type="button"
        value="Все"
      />
      <input
        className="active"
        onClick={[Function]}
        type="button"
        value="Активные"
      />
      <input
        className="blocked"
        onClick={[Function]}
        type="button"
        value="Заблокированные"
      />
    </div>
    <div
      className="mobileCompanyClients"
    >
      <div
        className="clientHeader"
      >
        <div
          className="clientSurname header"
        >
          Фамилия
        </div>
        <div
          className="clientName header"
        >
          Имя
        </div>
        <div
          className="clientPatronym header"
        >
          Отчество
        </div>
        <div
          className="clientBalance header"
        >
          Баланс
        </div>
        <div
          className="clientStatus header"
        >
          Статус
        </div>
        <div
          className="clientEdit header"
        >
          Редактировать
        </div>
        <div
          className="clientDelete header"
        >
          Удалить
        </div>
      </div>
      <div
        className="clients"
      >
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Иванов
          </div>
          <div
            className="clientName"
          >
            Иван
          </div>
          <div
            className="clientPatronym"
          >
            Иванович
          </div>
          <div
            className="clientBalance"
          >
            200
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Сидоров
          </div>
          <div
            className="clientName"
          >
            Сидор
          </div>
          <div
            className="clientPatronym"
          >
            Сидорович
          </div>
          <div
            className="clientBalance"
          >
            250
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Петров
          </div>
          <div
            className="clientName"
          >
            Пётр
          </div>
          <div
            className="clientPatronym"
          >
            Петрович
          </div>
          <div
            className="clientBalance"
          >
            180
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Григорьев
          </div>
          <div
            className="clientName"
          >
            Григорий
          </div>
          <div
            className="clientPatronym"
          >
            Григорьевич
          </div>
          <div
            className="clientBalance"
          >
            -220
          </div>
          <div
            className="clientStatus red"
          >
            blocked
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <input
    className="addClient"
    onClick={[Function]}
    type="button"
    value="добавить клиента"
  />
</div>
`;

exports[`MobileCompany add client should work 1`] = `
<div>
  <div
    className="mobileCompany"
  >
    <div
      className="buttons"
    >
      <input
        className="all"
        onClick={[Function]}
        type="button"
        value="Все"
      />
      <input
        className="active"
        onClick={[Function]}
        type="button"
        value="Активные"
      />
      <input
        className="blocked"
        onClick={[Function]}
        type="button"
        value="Заблокированные"
      />
    </div>
    <div
      className="mobileCompanyClients"
    >
      <div
        className="clientHeader"
      >
        <div
          className="clientSurname header"
        >
          Фамилия
        </div>
        <div
          className="clientName header"
        >
          Имя
        </div>
        <div
          className="clientPatronym header"
        >
          Отчество
        </div>
        <div
          className="clientBalance header"
        >
          Баланс
        </div>
        <div
          className="clientStatus header"
        >
          Статус
        </div>
        <div
          className="clientEdit header"
        >
          Редактировать
        </div>
        <div
          className="clientDelete header"
        >
          Удалить
        </div>
      </div>
      <div
        className="clients"
      >
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Иванов
          </div>
          <div
            className="clientName"
          >
            Иван
          </div>
          <div
            className="clientPatronym"
          >
            Иванович
          </div>
          <div
            className="clientBalance"
          >
            350
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Сидоров
          </div>
          <div
            className="clientName"
          >
            Сидор
          </div>
          <div
            className="clientPatronym"
          >
            Сидорович
          </div>
          <div
            className="clientBalance"
          >
            250
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Петров
          </div>
          <div
            className="clientName"
          >
            Пётр
          </div>
          <div
            className="clientPatronym"
          >
            Петрович
          </div>
          <div
            className="clientBalance"
          >
            180
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Григорьев
          </div>
          <div
            className="clientName"
          >
            Григорий
          </div>
          <div
            className="clientPatronym"
          >
            Григорьевич
          </div>
          <div
            className="clientBalance"
          >
            -220
          </div>
          <div
            className="clientStatus red"
          >
            blocked
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Антонов
          </div>
          <div
            className="clientName"
          >
            Антон
          </div>
          <div
            className="clientPatronym"
          >
            Антонович
          </div>
          <div
            className="clientBalance"
          >
            350
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <input
    className="addClient"
    onClick={[Function]}
    type="button"
    value="добавить клиента"
  />
</div>
`;

exports[`MobileCompany delete client should work 1`] = `
<div>
  <div
    className="mobileCompany"
  >
    <div
      className="buttons"
    >
      <input
        className="all"
        onClick={[Function]}
        type="button"
        value="Все"
      />
      <input
        className="active"
        onClick={[Function]}
        type="button"
        value="Активные"
      />
      <input
        className="blocked"
        onClick={[Function]}
        type="button"
        value="Заблокированные"
      />
    </div>
    <div
      className="mobileCompanyClients"
    >
      <div
        className="clientHeader"
      >
        <div
          className="clientSurname header"
        >
          Фамилия
        </div>
        <div
          className="clientName header"
        >
          Имя
        </div>
        <div
          className="clientPatronym header"
        >
          Отчество
        </div>
        <div
          className="clientBalance header"
        >
          Баланс
        </div>
        <div
          className="clientStatus header"
        >
          Статус
        </div>
        <div
          className="clientEdit header"
        >
          Редактировать
        </div>
        <div
          className="clientDelete header"
        >
          Удалить
        </div>
      </div>
      <div
        className="clients"
      >
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Иванов
          </div>
          <div
            className="clientName"
          >
            Иван
          </div>
          <div
            className="clientPatronym"
          >
            Иванович
          </div>
          <div
            className="clientBalance"
          >
            350
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Сидоров
          </div>
          <div
            className="clientName"
          >
            Сидор
          </div>
          <div
            className="clientPatronym"
          >
            Сидорович
          </div>
          <div
            className="clientBalance"
          >
            250
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Петров
          </div>
          <div
            className="clientName"
          >
            Пётр
          </div>
          <div
            className="clientPatronym"
          >
            Петрович
          </div>
          <div
            className="clientBalance"
          >
            180
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Григорьев
          </div>
          <div
            className="clientName"
          >
            Григорий
          </div>
          <div
            className="clientPatronym"
          >
            Григорьевич
          </div>
          <div
            className="clientBalance"
          >
            -220
          </div>
          <div
            className="clientStatus red"
          >
            blocked
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <input
    className="addClient"
    onClick={[Function]}
    type="button"
    value="добавить клиента"
  />
</div>
`;

exports[`MobileCompany edit client should work 1`] = `
<div>
  <div
    className="mobileCompany"
  >
    <div
      className="buttons"
    >
      <input
        className="all"
        onClick={[Function]}
        type="button"
        value="Все"
      />
      <input
        className="active"
        onClick={[Function]}
        type="button"
        value="Активные"
      />
      <input
        className="blocked"
        onClick={[Function]}
        type="button"
        value="Заблокированные"
      />
    </div>
    <div
      className="mobileCompanyClients"
    >
      <div
        className="clientHeader"
      >
        <div
          className="clientSurname header"
        >
          Фамилия
        </div>
        <div
          className="clientName header"
        >
          Имя
        </div>
        <div
          className="clientPatronym header"
        >
          Отчество
        </div>
        <div
          className="clientBalance header"
        >
          Баланс
        </div>
        <div
          className="clientStatus header"
        >
          Статус
        </div>
        <div
          className="clientEdit header"
        >
          Редактировать
        </div>
        <div
          className="clientDelete header"
        >
          Удалить
        </div>
      </div>
      <div
        className="clients"
      >
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Иванов
          </div>
          <div
            className="clientName"
          >
            Иван
          </div>
          <div
            className="clientPatronym"
          >
            Иванович
          </div>
          <div
            className="clientBalance"
          >
            350
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Сидоров
          </div>
          <div
            className="clientName"
          >
            Сидор
          </div>
          <div
            className="clientPatronym"
          >
            Сидорович
          </div>
          <div
            className="clientBalance"
          >
            250
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Петров
          </div>
          <div
            className="clientName"
          >
            Пётр
          </div>
          <div
            className="clientPatronym"
          >
            Петрович
          </div>
          <div
            className="clientBalance"
          >
            180
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Григорьев
          </div>
          <div
            className="clientName"
          >
            Григорий
          </div>
          <div
            className="clientPatronym"
          >
            Григорьевич
          </div>
          <div
            className="clientBalance"
          >
            -220
          </div>
          <div
            className="clientStatus red"
          >
            blocked
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <input
    className="addClient"
    onClick={[Function]}
    type="button"
    value="добавить клиента"
  />
</div>
`;

exports[`MobileCompany show active client should work 1`] = `
<div>
  <div
    className="mobileCompany"
  >
    <div
      className="buttons"
    >
      <input
        className="all"
        onClick={[Function]}
        type="button"
        value="Все"
      />
      <input
        className="active"
        onClick={[Function]}
        type="button"
        value="Активные"
      />
      <input
        className="blocked"
        onClick={[Function]}
        type="button"
        value="Заблокированные"
      />
    </div>
    <div
      className="mobileCompanyClients"
    >
      <div
        className="clientHeader"
      >
        <div
          className="clientSurname header"
        >
          Фамилия
        </div>
        <div
          className="clientName header"
        >
          Имя
        </div>
        <div
          className="clientPatronym header"
        >
          Отчество
        </div>
        <div
          className="clientBalance header"
        >
          Баланс
        </div>
        <div
          className="clientStatus header"
        >
          Статус
        </div>
        <div
          className="clientEdit header"
        >
          Редактировать
        </div>
        <div
          className="clientDelete header"
        >
          Удалить
        </div>
      </div>
      <div
        className="clients"
      >
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Иванов
          </div>
          <div
            className="clientName"
          >
            Иван
          </div>
          <div
            className="clientPatronym"
          >
            Иванович
          </div>
          <div
            className="clientBalance"
          >
            200
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Сидоров
          </div>
          <div
            className="clientName"
          >
            Сидор
          </div>
          <div
            className="clientPatronym"
          >
            Сидорович
          </div>
          <div
            className="clientBalance"
          >
            250
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Петров
          </div>
          <div
            className="clientName"
          >
            Пётр
          </div>
          <div
            className="clientPatronym"
          >
            Петрович
          </div>
          <div
            className="clientBalance"
          >
            180
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <input
    className="addClient"
    onClick={[Function]}
    type="button"
    value="добавить клиента"
  />
</div>
`;

exports[`MobileCompany show all client should work 1`] = `
<div>
  <div
    className="mobileCompany"
  >
    <div
      className="buttons"
    >
      <input
        className="all"
        onClick={[Function]}
        type="button"
        value="Все"
      />
      <input
        className="active"
        onClick={[Function]}
        type="button"
        value="Активные"
      />
      <input
        className="blocked"
        onClick={[Function]}
        type="button"
        value="Заблокированные"
      />
    </div>
    <div
      className="mobileCompanyClients"
    >
      <div
        className="clientHeader"
      >
        <div
          className="clientSurname header"
        >
          Фамилия
        </div>
        <div
          className="clientName header"
        >
          Имя
        </div>
        <div
          className="clientPatronym header"
        >
          Отчество
        </div>
        <div
          className="clientBalance header"
        >
          Баланс
        </div>
        <div
          className="clientStatus header"
        >
          Статус
        </div>
        <div
          className="clientEdit header"
        >
          Редактировать
        </div>
        <div
          className="clientDelete header"
        >
          Удалить
        </div>
      </div>
      <div
        className="clients"
      >
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Иванов
          </div>
          <div
            className="clientName"
          >
            Иван
          </div>
          <div
            className="clientPatronym"
          >
            Иванович
          </div>
          <div
            className="clientBalance"
          >
            200
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Сидоров
          </div>
          <div
            className="clientName"
          >
            Сидор
          </div>
          <div
            className="clientPatronym"
          >
            Сидорович
          </div>
          <div
            className="clientBalance"
          >
            250
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Петров
          </div>
          <div
            className="clientName"
          >
            Пётр
          </div>
          <div
            className="clientPatronym"
          >
            Петрович
          </div>
          <div
            className="clientBalance"
          >
            180
          </div>
          <div
            className="clientStatus green"
          >
            active
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Григорьев
          </div>
          <div
            className="clientName"
          >
            Григорий
          </div>
          <div
            className="clientPatronym"
          >
            Григорьевич
          </div>
          <div
            className="clientBalance"
          >
            -220
          </div>
          <div
            className="clientStatus red"
          >
            blocked
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <input
    className="addClient"
    onClick={[Function]}
    type="button"
    value="добавить клиента"
  />
</div>
`;

exports[`MobileCompany show blocked client should work 1`] = `
<div>
  <div
    className="mobileCompany"
  >
    <div
      className="buttons"
    >
      <input
        className="all"
        onClick={[Function]}
        type="button"
        value="Все"
      />
      <input
        className="active"
        onClick={[Function]}
        type="button"
        value="Активные"
      />
      <input
        className="blocked"
        onClick={[Function]}
        type="button"
        value="Заблокированные"
      />
    </div>
    <div
      className="mobileCompanyClients"
    >
      <div
        className="clientHeader"
      >
        <div
          className="clientSurname header"
        >
          Фамилия
        </div>
        <div
          className="clientName header"
        >
          Имя
        </div>
        <div
          className="clientPatronym header"
        >
          Отчество
        </div>
        <div
          className="clientBalance header"
        >
          Баланс
        </div>
        <div
          className="clientStatus header"
        >
          Статус
        </div>
        <div
          className="clientEdit header"
        >
          Редактировать
        </div>
        <div
          className="clientDelete header"
        >
          Удалить
        </div>
      </div>
      <div
        className="clients"
      >
        <div
          className="mobileClient"
        >
          <div
            className="clientSurname"
          >
            Григорьев
          </div>
          <div
            className="clientName"
          >
            Григорий
          </div>
          <div
            className="clientPatronym"
          >
            Григорьевич
          </div>
          <div
            className="clientBalance"
          >
            -220
          </div>
          <div
            className="clientStatus red"
          >
            blocked
          </div>
          <div
            className="clientEdit"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Редактировать"
            />
          </div>
          <div
            className="clientDelete"
          >
            <input
              onClick={[Function]}
              type="button"
              value="Удалить"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <input
    className="addClient"
    onClick={[Function]}
    type="button"
    value="добавить клиента"
  />
</div>
`;
